tests/quote_test.py

```diff
@@ -0,0 +1,106 @@
+"""Quote Tests"""
+
+import unittest
+from app import create_app
+from database.models import Quote, Author
+
+
+class QuoteTest(unittest.TestCase):
+    """Quote Test Class"""
+
+    def setUp(self):
```

Comment:
```suggestion
    def set_up(self):
```
Or is it must to use `setup`?

Reply:
yes it must be setUp

Reply:
Okay, there must be a `tearDown` method which can destroy things that are being created in `setUp` method. Can you check that and add it?